core/config.js

```diff
@@ -0,0 +1,58 @@
+Nuxeo = Nuxeo || {};
+Nuxeo.UI = Nuxeo.UI || {};
+Nuxeo.UI.config = Nuxeo.UI.config || {};
+const { config } = Nuxeo.UI;
+
+Object.assign(config, {
+  /**
+   * Returns a property for a given `path`. If it is not defined, `fallback` is returned instead.
+   * If `fallback` is defined, the method will try to convert the property value to the same data type of `fallback`.
+   */
+  get(path, fallback) {
+    let val = path.split('.').reduce((a, b) => a && a[b], this);
+    if (fallback !== undefined && typeof val !== typeof fallback) {
+      let type;
+      switch (typeof fallback) {
```

Comment:
@Gabez0r If the fallback is an object like [here](https://github.com/nuxeo/nuxeo-web-ui/pull/1218/files#diff-d60605c9b1864e6e61a7205e6c3ee8b7568187839dd0b2ea37b74c0189f7898eR96) we don't anything?

Reply:
You are right. I wanted to check on `type` but forgot about it. It should do something like:
`val = type && type(val) || val;`

Reply:
Sounds fair to me 👍 I was wondering if the type is `object` if we should do something like `JSON.parse()` 🤔

Reply:
@richardsd I have update the could, you please check if the suggestion I made makes sense? Thanks 🙏

Reply:
👍 I think it makes sense.

Reply:
> Sounds fair to me 👍 I was wondering if the type is object if we should do something like JSON.parse() 🤔, or maybe it is not really necessary.

That's a good question. I was assuming that we were just worried about "casting" types, which I think would only apply to primitive types, but we can think about other use cases for objects.